bip-0388.mediawiki

```diff
@@ -242,7 +242,7 @@ Common single-signature account patterns:
 Common multisig schemes:
 * <tt>wsh(multi(2,@0/**,@1/**))</tt> - SegWit 2-of-2 multisig, keys in order.
 * <tt>sh(sortedmulti(2,@0/**,@1/**,@2/**))</tt> - Legacy 2-of-3 multisig, sorted keys.
-* <tt>tr(musig(@0/**,@1/**))</tt> - MuSig2 2-of-2 in the taproot keypath
+* <tt>tr(musig(@0,@1)/**)</tt> - MuSig2 2-of-2 in the taproot keypath
```

Comment:
This does remove the contradiction between the example here being the same as the invalid policy example in line 313 below.

Is there further explanation somewhere about the statement in line 313 `"Derivation before aggregation is not allowed in wallet policies (despite being allowed in BIP-390)"`?